serving/pom.xml

```diff
@@ -292,7 +240,7 @@
     <dependency>
       <groupId>org.junit.jupiter</groupId>
       <artifactId>junit-jupiter</artifactId>
-      <version>RELEASE</version>
+      <version>5.5.2</version>
```

Comment:
This quiets a Maven warning because the `RELEASE` meta version is deprecated. I'm not sure if this should be a 4.x version for JUnit 4, but this is the latest release version. Actually this dependency appears unused except in the Java SDK where it is re-declared since it isn't currently a child module.

Reply:
Yes, we're trying to see if we can use JUnit 5 test framework because testing for exceptions and parameterizing test cases is cleaner in JUnit 5. But since in Core, Serving and Ingestion there are some dependencies that may not be as quick to migrate the test to JUnit 5, currently we still use JUnit 4 for those.